### PR TITLE
feat: changes required to run on Ubuntu 20 agents

### PR DESCRIPTION
**Pull request by ernestojeda**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/edgexfoundry/ci-management/blob/main/.github/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Added labels
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Sandbox Testing
Test Links :

sample-service test:
https://jenkins.edgexfoundry.org/job/edgexfoundry/job/sample-service/view/change-requests/job/PR-141/7/console

device-coap-c
https://jenkins.edgexfoundry.org/job/edgexfoundry/job/device-coap-c/view/change-requests/job/PR-36/2/console

### CODE REVIEW

**Review by bill-mahoney — APPROVED** [review]
LGTM

File: vars/edgeXInfraShipLogs.groovy [hunk 1]
```diff
@@ -30,13 +30,10 @@ def call(body) {
...
     // running this inside the lftools container to avoid the 1-3 minute install of lftools
     // Dockerfile for this image can be found here: https://github.com/edgexfoundry/ci-build-images/blob/lftools/Dockerfile.logs-publish
-    docker.image("${env.DOCKER_REGISTRY}:10003/edgex-lftools-log-publisher:latest").inside('--privileged -u 0:0 -v /var/log/sa:/var/log/sa-host') {
+    docker.image("${env.DOCKER_REGISTRY}:10003/edgex-lftools-log-publisher:latest").inside('--privileged -u 0:0 -v /var/log/sysstat:/var/log/sysstat') {
         // A conversion needs to take place for the sysstat files (sar is called during the lftools deploy logs)
         // See: https://serverfault.com/questions/757771/how-to-read-sar-file-from-different-ubuntu-system
 
-        sh 'mkdir -p /var/log/sa'
-        sh 'for file in `ls /var/log/sa-host`; do sadf -c /var/log/sa-host/${file} > /var/log/sa/${file}; done'
-
```
Comment (soda480) [inline vars/edgeXInfraShipLogs.groovy:39]: no conversion required for sysstat i take it?

**Review by soda480 — APPROVED** [review]
LGTM, but just had one question.

File: vars/edgeXInfraShipLogs.groovy (hunk 1 above)
Comment (ernestojeda) [reply vars/edgeXInfraShipLogs.groovy:39]: No, so much easier. This was a pain to make work on CentOS

**Pull request merged**